### PR TITLE
Fix for the uuid_media_reneg API bug which prevents audio to be added to video-only call (bug #1862)

### PR DESCRIPTION
**Pull request by figaro2015**

### CODE REVIEW

File: src/switch_core_media.c [hunk 1]
```diff
@@ -13126,6 +13126,11 @@ SWITCH_DECLARE(switch_status_t) switch_core_media_receive_message(switch_core_se
 													   "codec_string", "=%s", switch_channel_get_variable(session->channel, "ep_codec_string"));
 				}
 				
+				// Required fix to allow audio to be added to video
```
[the remark this answers is not in the document]
Comment (figaro2015) [reply src/switch_core_media.c]: Done!

File: src/switch_core_media.c [hunk 2]
```diff
@@ -13126,6 +13126,12 @@ SWITCH_DECLARE(switch_status_t) switch_core_media_receive_message(switch_core_se
 													   "codec_string", "=%s", switch_channel_get_variable(session->channel, "ep_codec_string"));
 				}
 				
+				/* Required fix to allow audio to be added to video
```
Comment (andywolk) [inline src/switch_core_media.c:13129]: `Required fix` sounds a bit off. If merged there is no issue to fix anymore.

**Review by andywolk — CHANGES_REQUESTED** [review]

File: src/switch_core_media.c (hunk 2 above)
Comment (figaro2015) [reply src/switch_core_media.c:13129]: so, what exactly do you want me to change?

Comment (figaro2015) [reply src/switch_core_media.c]: Updated misleading comment

**Pull request closed without merging**